docs/docs/1.2.3/installation/helm.md

```diff
@@ -27,7 +27,6 @@ Once the repo is added, all following updates can be fetched with `helm repo upd
 At this point we can install and run Kuma using the following commands. We could use any Kubernetes namespace to install Kuma, by default we suggest using `kuma-system`:
 
 ```sh
-$ kubectl create namespace kuma-system
 $ helm install --namespace kuma-system kuma kuma/kuma
```

Comment:
```suggestion
$ helm install --create-namespace --namespace kuma-system kuma kuma/kuma
```
Maybe this is what we want instead?

Reply:
This seems correct indeed.
Seems like I misunderstood something when we talked about it with @bartsmykla.
@bartsmykla  can you double check that what @austince is saying matches what you said earlier?

Reply:
it's exactly what I was thinking about